solid-b4c-rods/README.md

```diff
@@ -0,0 +1,28 @@
+# TAP MSR
+
+This directory contains a Serpent 2 Model for the Transatomic Power (TAP) Molten Salt Reactor (MSR).
+The model is based of the technical report published by Oak Ridge National Laboratory (ORNL), which can be found here https://www.osti.gov/biblio/1410921
+
+Several assumptions have been made due to lack of information.
+The density of the salt was determined using $`\rho(T)=6.105-0.001272T[K]`$
+which is based on https://aip.scitation.org/doi/10.1063/1.3253134 although the temperatures of interest are outside the support of the equation.
+
+The control rods are solid, made of B4C with reactivity worth 3.9%. Control rods are fully withdrawn (see lines 7-14 tap.ini).
+
+45 degrees symmetry with reflective boundary condition was used.
+
+10000 neutrons, 1000 active cycles, 100 inactive cycles gave STD+/-22pcm
+9 minutes
```

Comment:
1 core?

Reply:
12 threads, 6 cores. Added this in README